### DISCUSSION TITLE
不知道是不是下载的版本问题, 重启后所有设定都初始化了

### DISCUSSION THREAD

**Discussion by IzumoNoOkuni** [immortalwrt/immortalwrt#743] [Q&A] [answered]
版本是21.02.1/targets/x86/64/generic-image-efi.iso
PVE下虚拟机运行的, 进入系统后重新设定ip地址然后reboot, ip地址还是没被修改, 通过uci命令修改lan口ip可以成功, 但是reboot之后又全部回复出厂的设置了.
```
uci set network.lan.ipaddr='192.168.2.1'
uci commit
/etc/init.d/network restart
```

**Comment by 1715173329** [comment, marked as answer]
iso 文件是 live cd。请下载 qcow2 等格式的镜像。

**Comment by TnZzZHlp** [comment]
google一下pve安装openwrt

**Reply by IzumoNoOkuni** [reply]
那肯定有查过, 网上大部分教程都是LEDE的镜像, 然后都是img格式.
